zerver/lib/url_encoding.py

```diff
@@ -74,15 +89,17 @@ def near_stream_message_url(realm: Realm, message: Dict[str, Any]) -> str:
         "topic",
         encoded_topic,
         "near",
-        message_id,
+        message_id_,
     ]
     full_url = "/".join(parts)
     return full_url
 
 
-def near_pm_message_url(realm: Realm, message: Dict[str, Any]) -> str:
-    message_id = str(message["id"])
-    str_user_ids = [str(recipient["id"]) for recipient in message["display_recipient"]]
+def near_pm_message_url(
+    realm: Realm, message_id: int, display_recipient: List[UserDisplayRecipient]
+) -> str:
+    msg_id = str(message_id)
```

Comment:
We have a profile policy of avoiding abbreviations, and also not using two similar variables like this. I would do `message_id_string = str(message_id)` here.

Reply:
That said, are these variables even necessary? I'd Kinda have expected these integers to just get coerced to strings anyway when joining...

Reply:
If I don´t convert them to string, I get a warning from mypy!